tests/upgrade/raycluster_sdk_upgrade_test.py

```diff
@@ -0,0 +1,154 @@
+import requests
+from time import sleep
+
+from torchx.specs.api import AppState, is_terminal
+
+from codeflare_sdk import Cluster, ClusterConfiguration, TokenAuthentication
+from codeflare_sdk.job import RayJobClient
+
+import pytest
+
+from tests.e2e.support import *
+from codeflare_sdk.cluster.cluster import get_cluster
+
+namespace = "test-ns-rayupgrade"
+
+
+# Creates a Ray cluster
+class TestMNISTRayClusterUp:
+    def setup_method(self):
```

Comment:
Perhaps we could include a teardown as well to delete the namespace after the test is ran, something like:
```
    def teardown_method(self):
        delete_namespace(self)
```

Reply:
Not in this case as the test is intended to verify upgrade.
So the intended use case is to create a cluster, upgrade operator and/or SDK, get cluster and make sure we still can work with it.

Reply:
As a part of upgrade test, we need namespace and raycluster created during `TestRayClusterUp` to be used by `Test_mnist_job_submission`, right? 
If we include this tear-down method in `TestRayClusterUp`, then the created namespace will be deleted and won't be available for `Test_mnist_job_submission`